.github/workflows/buildsCI.yaml

```diff
@@ -332,7 +332,7 @@ jobs:
           cmake --build . --config $LRS_BUILD_CONFIG -- -j4
           cd ../wrappers/nodejs/ && npm install && cd -
 
-
+    
```

Comment:
Please undo.
You should always take care not to add redundant changes in other files.
When we look at the history we prefer to see real changes and who made them :)

Reply:
Noted, I will.
FYI, sublime warning me about unnecessary spaces in this file...

Reply:
OK, when you edit a file/section you can align it.
sublime might be configured differently for every user with regards to spaces.. so better not let it change it for you